### PR TITLE
Summary of the steps

### PR DESCRIPTION
**Pull request by dinemamadou**
Testing: fork, clone and added new links to the README.md file.

### CODE REVIEW

File: misc/links/README.md [hunk 1]
```diff
@@ -4,11 +4,4 @@ Useful links
...
 - [GitHub](www.github.com)
 - [Rutgers](www.rutgers.edu)
-- [this is a link](www.github.com)
-- [Personal web](crislozano.weebly.com)
-- [My awesome link](coakden.github.io)
-- [Hello! English](www.helloenglishmadrid.com)
-- [my awesome link](http://rci.rutgers.edu/~erb102/)
-- [Reddit](www.reddit.com)
-- [DataSci](http://www.jvcasillas.com/ru_teaching/ru_spanish_589/589_01_s2018/)
-- [Sakai](https://sakai.rutgers.edu/portal)
```
Comment (jvcasillas) [inline misc/links/README.md]: There is no need to delete all of the previous links. Just add your personal website.

File: misc/students/firstname_lastname/README.md [hunk 2]
```diff
@@ -1,9 +0,0 @@
-Firstname Lastname's dropbox
-============================
-
-This is an example. 
-
-- Name: Firstname Lastname 
-- Email: my_email@email.com
-- Website: www.github.com
-- Goal: To learn R
```
Comment (jvcasillas) [inline misc/students/firstname_lastname/README.md]: Please leave this file. It serves as an example for other students.

File: misc/students/firstname_lastname/summaries/wickham_2015_summary.md [hunk 3]
```diff
@@ -1,5 +0,0 @@
-# Teaching Safe-Stats, Not Statistical Abstinence
-
-## Wickham (2015) summary
-
-Four sentence summary goes here.
```
Comment (jvcasillas) [inline misc/students/firstname_lastname/summaries/wickham_2015_summary.md]: Please leave this as is. It serves as an example for other students.

File: misc/links/README.md (hunk 1 above)
Comment (dinemamadou) [reply misc/links/README.md]: How do I revert these changes?

**Pull request closed without merging**